tests/test_objective_funs.py

```diff
@@ -3673,13 +3675,22 @@ def test_objective_print_widths():
 def test_objective_docstring():
     """Test that the objective docstring and collect_docs are consistent."""
     objective_docs = _Objective.__doc__.rstrip()
-    doc_header = (
-        "Objective (or constraint) used in the optimization of an Equilibrium.\n\n"
-        + "    Parameters\n"
-        + "    ----------\n"
-        + "    things : Optimizable or tuple/list of Optimizable\n"
-        + "        Objects that will be optimized to satisfy the Objective.\n"
-    )
+    if Version(platform.python_version()) >= Version("3.13"):
```

Comment:
Why is this needed if the docs are still built with an older version of Python?

And will we need to reformat docstrings everywhere if we want to use newer python for building docs?